integration-tests/features/steps/fsfreeze.py

```diff
@@ -50,13 +50,12 @@ def test_migrate_fsfreeze(context, source, target, not_frozen=None):
 
         _replace_fsfreeze_bin(context, source)
```

Comment:
this does not look like a cool idea to directly alter a binary on the source server, though maybe i got something wrong